content/en/blog/2022/meet-our-contributors-APAC-China-region-03.md

```diff
@@ -0,0 +1,70 @@
+---
+layout: blog
+title: "Meet Our Contributors - APAC (China region)"
+date: 2022-08-15
+slug: meet-our-contributors-china-ep-03
+canonicalUrl: https://www.kubernetes.dev/blog/2022/07/11/meet-our-contributors-chn-ep-03/
```

Comment:
```suggestion
```
You set a `canonicalUrl` on the **mirror** of the article, and you don't set this field on the original.